downstream/titles/release-notes/topics/errata-24-aug-10.adoc

```diff
@@ -0,0 +1,101 @@
+// This is the release notes file for AAP 2.4 errata bundle dated August 10 2023.
+
+= {PlatformNameShort} 2.4 - August 10, 2023
+
+link:https://access.redhat.com/errata/RHBA-2023:4621[Red Hat Errata Release]
+
+== {ControllerNameStart}
+
+.New features and enhancements
+
+* automation-controller has been updated to 4.4.1.
+
+.Security fixes
+
+* automation controller: Html injection in custom login info (CVE-2023-3971)
+
+.Bug fixes
+
+* Organization admin users are no longer shown an error on the Instances list.
+
+* Fixed the workflow job within workflow approval to display the correct details.
+
+* Credential name search in the ad hoc commands prompt no longer requires case-sensitive input.
+
+* The *Back to list* button in the controller UI now maintains previous search filters.
+
+* *Topology view* and *Instances* are only available as sidebar menu options to System Administrators and System Auditors.
+
+* Fixed the frequency of the scheduler to run on the correct day of the week as specified by the user.
+
+* Fixed an issue with slow database UPDATE statements when using nested tasks (include_tasks) causing task manager timeout.
+
+* Added a setting to enable the queuing for Rsyslog to handle higher work volumes (`LOG_AGGREGATOR_ACTION_MAX_DISK_USAGE_GB`).
+
+* Added the ability to add execution and hop nodes to VM-based controller installations from the UI.
+
+* Added the `awx-manage` command for creating future events table partitions.
+
+* Re-enabled Pendo support by providing the correct Pendo API key.
+
+* Added the ability to filter teams by using partial names in the dialog for granting teams access to a resource.
+
+* Fixed a bug where a weekly rrule string without a BYDAY value would result in the UI throwing a TypeError.
```

Comment:
Just a line edit here, "rrule" should be "rule".

Reply:
I may be wrong but my understanding from reading the Jira ticket is that rrule is a specific attribute (I'll send you on a screenshot of the ticket)